src/test/resources/features/ifs/GetDebtForDrierCase.feature

```diff
@@ -0,0 +1,149 @@
+    When the debt item is sent to the ifs service
+    Then the ifs service will respond with
+      | dailyInterest | totalInterest | intRate | totalAmountToPay | totalAmountWithInterest | numberChargeableDays |
+      | 0             | 0             | 0       | 5000             | 5000                    | 0                    |
+#
+  Scenario: DRIER debt Zero Amount Edge Case
+    Given a debt item
+      | amount | dateAmount | dateCalculationTo | regime | chargeType | interestBearing |
+      | 0      | 2021-03-01 | 2021-03-08        | DRIER  | NI         | true            |
+    When the debt item is sent to the ifs service
+    Then the ifs service will respond with
+      | dailyInterest | totalInterest | intRate | totalAmountToPay | totalAmountWithInterest | numberChargeableDays |
+      | 0             | 0             | 1       | 0                | 0                       | 8                    |
+
+#Below scenario currently fails as api returns daily interest of -0.0001. Should negative amounts be possible?
+  @ignore
+  Scenario: DRIER debt Amount is negative (Edge Case)
+    Given a debt item
+      | amount | dateAmount | dateCalculationTo | regime | chargeType | interestBearing |
+      | -1     | 2021-03-01 | 2021-03-08        | DRIER  | NI         | true            |
+    When the debt item is sent to the ifs service
+    Then the ifs service will respond with
+      | dailyInterest | totalInterest | intRate | totalAmountToPay | totalAmountWithInterest | numberChargeableDays |
+      | 0             | 0             | 1       | 0                | 0                       | 8                    |
+
+  Scenario: DRIER debt Amount non integer (Edge Case)
+    Given a debt item
+      | amount | dateAmount | dateCalculationTo | regime | chargeType | interestBearing |
+      | n      | 2021-03-01 | 2021-03-08        | DRIER  | NI         | true            |
+    When the debt item is sent to the ifs service
+    Then the ifs service will respond with '/amount' missing or invalid
+
+#    Below scenario currently fails. api should not accept decimal places. DTD-191 to fix this
+  @ignore
+  Scenario: DRIER debt Amount non integer (Edge Case)
+    Given a debt item
+      | amount | dateAmount | dateCalculationTo | regime | chargeType | interestBearing |
+      | 1.0    | 2021-03-01 | 2021-03-08        | DRIER  | NI         | true            |
+    When the debt item is sent to the ifs service
+    Then the ifs service will respond with '/amount' missing or invalid
+
+  Scenario: DRIER debt empty Amount (Edge Case)
+    Given a debt item
+      | amount | dateAmount | dateCalculationTo | regime | chargeType | interestBearing |
+      |        | 2021-03-01 | 2021-03-08        | DRIER  | NI         | true            |
+    When the debt item is sent to the ifs service
+    Then the ifs service will respond with '/amount' missing or invalid
+
+  Scenario: DRIER debt invalid entry in Date Amount (Edge Case)
```

Comment:
In the scenario description, "Date Amount" doesn't make sense. Please change to "Date value"